### DISCUSSION TITLE
docker manifest not found [May 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5784] [Discord Threads]
> 🙋‍♂️ **cristianetiquetas**

**Comment by o1lab** [comment]
> 🙋‍♂️ **wkw**  
#### just tested and it s working fine. can u show the command you used?  
#  
> 🙋‍♂️ **cristianetiquetas**  
#### Hello, u used the command from website  
#  
> 🙋‍♂️ **cristianetiquetas**  
#### I used  
#  
> 🙋‍♂️ **cristianetiquetas**  
#### CentOS 7 version  
#  
> 🙋‍♂️ **cristianetiquetas**  
#### docker run -d --name nocodb \   
#### -v "$(pwd)"/nocodb:/usr/app/data/ \   
#### -p 8080:8080 \   
#### nocodb/nocodb:latest  
#  
> 🙋‍♂️ **cristianetiquetas**  
#### my Docker version 19.03.9, build 9d988398e7  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1099819908939927744)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
